app/views/dashboard/_activity.html.erb

```diff
@@ -20,54 +20,54 @@
       <span class="caret"></span>
     </button>
 
-    <div class="dropdown-menu">
-      <a class="form-check dropdown-item" href="#">
+    <ul class="dropdown-menu">
+      <li class="form-check dropdown-item">
         <label class="form-check-label">
-          <input data-type="all" class="node-type-all" type="checkbox" checked="checked" /> 
+          <input data-type="all" class="node-type-all form-check-input" type="checkbox" checked="checked" /> 
           <%= t('dashboard._activity.dropdown.all') %>
         </label>
-      </a>
+      </li>
 
       <div class="dropdown-divider"></div>
 
-      <a class="form-check dropdown-item" href="#">
+      <li class="form-check dropdown-item">
         <label class="form-check-label">
-          <input data-type="note" class="node-type node-type-note" type="checkbox" checked="checked" /> 
+          <input data-type="note" class="node-type node-type-note form-check-input" type="checkbox" checked="checked" /> 
           <%= t('dashboard._activity.dropdown.research_notes') %>
         </label>
-      </a>
-      <a class="form-check dropdown-item" href="#">
+      </li>
+      <li class="form-check dropdown-item">
         <label class="form-check-label">
-          <input data-type="question" class="node-type node-type-question" type="checkbox" checked="checked" /> 
+          <input data-type="question" class="node-type node-type-question form-check-input" type="checkbox" checked="checked" /> 
           <%= t('dashboard._activity.dropdown.question') %>
         </label>
-      </a>
-      <a class="form-check dropdown-item" href="#">
+      </li>
+      <li class="form-check dropdown-item">
         <label class="form-check-label">
-          <input data-type="event" class="node-type node-type-event" type="checkbox" checked="checked" /> 
+          <input data-type="event" class="node-type node-type-event form-check-input" type="checkbox" checked="checked" /> 
           <%= t('dashboard._activity.dropdown.events') %>
         </label>
-      </a>
-      <a class="form-check dropdown-item" href="#">
+      </li>
+      <li class="form-check dropdown-item">
         <label class="form-check-label">
-          <input data-type="comment" class="node-type node-type-comment" type="checkbox" checked="checked" /> 
+          <input data-type="comment" class="node-type node-type-comment form-check-input" type="checkbox" checked="checked" /> 
           <%= t('dashboard._activity.dropdown.comments') %>
         </label>
-      </a>
+      </li>
 
-      <a class="d-lg-none dropdown-item" href="#">
```

Comment:
The wiki link is set to hide on large screens. I don't know why so I left it as it was.

Reply:
hey @michelelong, on larger screens, you'll notice, we show a sidebar which has wiki tab in it and that shows all the updates of wiki edits. But, in mobile view or in smaller screens, we can't have a sidebar so we show wiki edit updates here too.

Reply:
I see now, thanks. Updated the wiki checkbox classes for mobile.